data/helpers.d/filesystem

```diff
@@ -98,29 +98,20 @@ ynh_store_file_checksum () {
 #
 # $app should be defined when calling this helper
 #
-# usage: ynh_backup_if_checksum_is_different file [compress]
+# usage: ynh_backup_if_checksum_is_different file
 # | arg: file - The file on which the checksum test will be perfomed.
-# | arg: compress - 1 to compress the backup instead of a simple copy
-# A compression is needed for a file which will be analyzed even if its name is different.
 #
 # | ret: Return the name a the backup file, or nothing
 ynh_backup_if_checksum_is_different () {
 	local file=$1
-	local compress_backup=${2:-0}	# If $2 is empty, compress_backup will set at 0
 	local checksum_setting_name=checksum_${file//[\/ ]/_}	# Replace all '/' and ' ' by '_'
 	local checksum_value=$(ynh_app_setting_get $app $checksum_setting_name)
 	if [ -n "$checksum_value" ]
 	then	# Proceed only if a value was stored into the app settings
 		if ! echo "$checksum_value $file" | sudo md5sum -c --status
 		then	# If the checksum is now different
-			backup_file="$file.backup.$(date '+%d.%m.%y_%Hh%M,%Ss')"
-			if [ $compress_backup -eq 1 ]
-			then
-				sudo tar --create --gzip --file "$backup_file.tar.gz" "$file"	# Backup the current file and compress
-				backup_file="$backup_file.tar.gz"
-			else
-				sudo cp -a "$file" "$backup_file"	# Backup the current file
-			fi
+			backup_file="/home/yunohost.conf/backup$file.backup.$(date '+%d.%m.%y_%Hh%M,%Ss')"
```

Comment:
I don't think so, because $file should be an absolute path.
But, in fact, I didn't test this code... (shame on me, I know...)

But, to prevent a error, if a double slash it's not a problem. It's better than nothing.